freebsd sh's `command` doesn't accept multiple command names to search for and barfs with
"command: wrong number of arguments"
diff --git a/tests/compare.sh b/tests/compare.sh
index 0943ea8..6d715ec 100755
--- a/tests/compare.sh
+++ b/tests/compare.sh
@@ -25,7 +25,7 @@ test -e "${referencefile}" || usage "No such file: '${referencefile}'"
 test -x "${mysofa2json}" || usage "No mysofa2json executable: '${mysofa2json}'"
 
 SCRIPTDIR=${0%/*}
-NODEJS=$(command -v node nodejs false | head -1)
+NODEJS=node
 TMP1="$(mktemp -p . tmp1-XXXXXXXX.json)"
 TMP2="$(mktemp -p . tmp2-XXXXXXXX.txt)"
 TMP3="$(mktemp -p . tmp3-XXXXXXXX.json)"
diff --git a/tests/compareIgnoreNew.sh b/tests/compareIgnoreNew.sh
index 9fc5e5d..6aed5a5 100755
--- a/tests/compareIgnoreNew.sh
+++ b/tests/compareIgnoreNew.sh
@@ -25,7 +25,7 @@ test -e "${referencefile}" || usage "No such file: '${referencefile}'"
 test -x "${mysofa2json}" || usage "No mysofa2json executable: '${mysofa2json}'"
 
 SCRIPTDIR=${0%/*}
-NODEJS=$(command -v node nodejs false | head -1)
+NODEJS=node
 TMP1="$(mktemp -p . tmp1-XXXXXXXX.json)"
 TMP2="$(mktemp -p . tmp2-XXXXXXXX.txt)"
 TMP3="$(mktemp -p . tmp3-XXXXXXXX.json)"
